pkgs/tools/text/termbook/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform }:
```

Comment:
```suggestion
{ lib, rustPlatform, fetchFromGitHub, stdenv, darwin }:
```

Reply:
why rustPlatform first and stdenv last? Is there a convention regarding the order?

Reply:
> [Arguments should be listed in the order they are used, with the exception of `lib`, which always goes first.](https://nixos.org/manual/nixpkgs/unstable/#sec-syntax)

it's kind of burried in the manual